core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationLoader.java

```diff
@@ -27,11 +28,24 @@
 
     private static final Logger logger = Log.getLogger();
 
+    /**
+     * Do not use - only for test purposes
+     *
+     * @return A {@link Configuration} instance
+     */
     public static Configuration load() {
```

Comment:
It's used in a lot of test classes, so having it package protected wouldn't solve it.
The reason why I put it there is to avoid using this method in our internal implementation as it has already happened a few times - it needs to be used a path to the project directory provided by ST API otherwise it won't work in fork configuration nor in other integration implementations.
I'd love to remove it, but I'm not sure how to do it... maybe just rewrite all the usages in tests classes. I'll create an issue for it as a separated task.